File: Maslo8/src/main/java/SkiLift.java
import Lab.GazolineType;

public class SkiLift extends Technique {
    private double liftingWeight;
    private double distanceToCover;
    public TechniqueEnum type = TechniqueEnum.SKILIFT;
    public Wheel wheel = Wheel.METAL;

    public double getLiftingWeight() {

        return liftingWeight;
    }

    public SkiLift() {
    }

    @Override
    public String getHeaders() {
        return super.getHeaders();
    }

    @Override
    public String toCSV() {
        return super.toCSV();
    }

    public String toString() {
        return super.toString() + "; Lift weight " + this.liftingWeight + "; Distanceto cover " + this.distanceToCover;
    }

    public SkiLift(String name, int amount, int run, double liftingWeight,
                   double distanceToCover, TechniqueEnum type, GazolineType gazolineType, Wheel wheel, final Integer id) {
        super(name, amount, run, type, wheel, gazolineType, id);
        this.liftingWeight = liftingWeight;
        this.distanceToCover = distanceToCover;


    }

    public void setLiftingWeight(double liftingWeight) {
        this.liftingWeight = liftingWeight;
    }

    public double getDistanceToCover() {
        return distanceToCover;
    }

    public void setDistanceToCover(double distanceToCover) {
        this.distanceToCover = distanceToCover;
    }


}

File: Lab2/ua/com/lviv/iot/SkiLift.py
from ua.com.lviv.iot.GazolineType import Gaz
from ua.com.lviv.iot.Technique import *
from ua.com.lviv.iot.WheelType import Wheel


class SkiLift(Technique):
    def __init__(self, name="unknown", amount=0, run=0):
        super().__init__(name, amount,run)
        self.type = Types.SKILIFT.name
        self.wheel = Wheel.WOODEN.name
        self.gazolineType = Gaz.A95.name
